<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginning</class>
 <widget class="QDialog" name="loginning">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>574</width>
    <height>423</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>574</width>
    <height>423</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вход/Регистрация</string>
  </property>
  <widget class="QLabel" name="loginningLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вход/Регистрация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="psswdLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmationPsswdLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Подтвердите пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>97</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signInButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>97</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>97</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
